Apps/WebClient/src/ClientApp/src/models/requestResult.ts

```diff
@@ -13,4 +13,6 @@ export default interface RequestResult<T> {
     resultStatus: ResultType;
     // The message associated to the request (could be empty)
     resultMessage: string;
+    // The error code  associated to the request (could be empty)
+    errorCode: string;
```

Comment:
can we initialize errorCode: string = ""; here instead of having to set it to "" empty every where i.e. in RestImmunizationService, RestLaboratoryService etc... below?

Reply:
well this is an interface and does not take the value initialization